OAuthdSetup.md

```diff
@@ -1,5 +1,12 @@
 ### Setting up oauthd daemon locally
 
+This service is installed separately and will get integrated with the project by updating the environment variables in `.env` file by updating `EACT_APP_OAUTHD_KEY` and `EACT_APP_OAUTHD_URL`
```

Comment:
Correct the spelling mistake in environment variables, it's `REACT_APP_OAUTHD_KEY` and `REACT_APP_OAUTHD_URL`

Reply:
Mention the Issue #544 also that you're fixing in the commit message or in title of PR